windows/.gitignore

```
@@ -10,7 +10,7 @@
 [Dd]ebugPublic/
 [Rr]elease/
 [Rr]eleases/
-**/AppPackages/**
+AppPackages/**
```

Nit: Afaik, `.gitignore` does pattern matching so we don't need the asterisks at all:
```suggestion
AppPackages/
```